src/create-route-map.js

```diff
@@ -43,6 +43,17 @@ function addRouteRecord (
   }
 
   if (route.children) {
+    // Warn if route is named and has a default child route.
+    // If users navigate to this route by name, the default child will
+    // not be rendered (GH Issue #629)
+    if (process.env.NODE_ENV !== 'production') {
```

Comment:
I was thinking about this: https://github.com/rollup/rollup-plugin-replace
It has the same behaviour that is already used in other repositories like the wepback template